home/dot_config/mackup/.mackup/edge.cfg

```diff
@@ -0,0 +1,10 @@
+[application]
+name = Edge
+
+[configuration_files]
+{{- if (eq .host.distro.family "darwin") -}}
+Library/Application Support/Microsoft Edge/Default
+{{- else if (eq .host.distro.family "linux") -}}
+.config/microsoft-edge/Default
```

Comment:
Shouldn't this be under `[xdg_configuration_files]`? See the bottom of this file: https://github.com/lra/mackup/tree/master/doc

Please go through all these and make sure they follow that format.